src/functional.py

```diff
@@ -1,7 +1,5 @@
 import nipype.interfaces.io as nio           # Data i/o 
```

Comment:
this is a better shortcut
```suggestion
import nipype.interfaces.io as no           # Data i/o 
```